doc/api/fs.md

```diff
@@ -2529,6 +2529,19 @@ changes:
 Asynchronous rename(2). No arguments other than a possible exception are given
 to the completion callback.
 
+This function will take the old path name of a file and rename it to the path
+name provided in the second parameter.
+
+Here is an example below (assuming the file "firstfile.txt" already exists):
+
+```js
+fs.rename('firstfile.txt', 'secondfile.txt', (err) => {
+	if(err) throw err;
```

Comment:
Also `if(err)` -> `if (err)` (with a space).

Sorry for the nit. Our doc examples are linted and these tiny things break the linting.

BTW, you can check your doc examples with this command from the repository root:
```console
node tools/node_modules/eslint/bin/eslint.js --rulesdir tools/eslint-rules/ --ext=.md doc
```